cli/cmd/ensureDeps.go

```
@@ -0,0 +1,330 @@
+		return fmt.Errorf("failed to extract asdf: %w", err)
+	}
+
+	// Add sourcing lines to .bashrc/.zshrc (basic)
+	shellRcFile, err := detectShellRCFile()
+	if err != nil {
+		return fmt.Errorf("failed to detect shell RC file: %w", err)
+	}
+
+	f, _ := os.OpenFile(shellRcFile, os.O_APPEND|os.O_WRONLY, 0644)
+	defer f.Close()
+	f.WriteString("\n\nexport PATH=\"${ASDF_DATA_DIR:-$HOME/.asdf}/shims:$PATH\"\n")
```

golangci-lint (1.64.8)

Error return value of `f.WriteString` is not checked

(errcheck)